samples/msal-node-samples/auth-code-cli-app/src/authConfig.js

```diff
@@ -0,0 +1,26 @@
+const { LogLevel } = require("@azure/msal-node");
+
+const cacheLocation = "./src/data/cache.json";
+const cachePlugin = require('../../cachePlugin')(cacheLocation);
+
+module.exports = {
+    authConfig: {
+        auth: {
+            clientId: "c3a8e9df-f1d4-427d-be73-acab139c40fd",
```

Comment:
Use a zero guid or some other indicator that they have to enter something here.  Comment that this needs to be updated here would be more useful than what you have on Index.js, line 10

Reply:
+1

Reply:
Done